### ISSUE TITLE
open review

### ISSUE THREAD

**Issue by WangYipu2002** [zjunlp/Deco#8]
We have seen your work published in ICLR2025. since the review comments for that conference are public, we have looked at your review comments. We found that it does not show your rebuttal to the review. can you provide a response to the review comments.

**Comment by zxlzr** [comment]
Hello, thank you for your interest in our work. We have submitted a rebuttal, but we’re not sure why it’s not publicly visible. If you have any specific questions, feel free to ask directly.

**Comment by WangYipu2002** [comment]
Thank you for your reply, I hold similar doubts about these three questions given by the reviewer, can you provide some explanations to help me understand this work better?


1.For each time step t, language tokens that are not related to the visual input but are essential for sentence generation may be influenced as they pass through the proposed method. There appears to be a lack of investigation into the nature of this influence.
2.Could you provide more evidence to demonstrate how dynamic soft modulation prevents abrupt changes in logits? Additional ablation studies might further substantiate this claim.
3.POPE evaluates answers in Yes/No. How could decoding strategy have impact on the performance for this benchmark?

**Comment by sunnychenxiwang** [comment]
We apologize for not making the rebuttal process public. Here, we provide our responses to these questions:
1. To balance visually relevant tokens and visually irrelevant tokens, the soft modulation in the DeCo process plays a crucial role. In our practical experiments, we find that tokens unrelated to vision tend to have lower probabilities in the preceding layers. To minimize the impact on the original response style, we introduce the dynamic modulation coefficient. We apologize for the difficulty in conducting statistical experiments given the diversity of generated tokens. If you are particularly interested, we provide additional cases of soft modulation analysis in our paper, as shown in Figure 13. Soft modulation mitigates the excessive alteration of the original generated response style caused by abrupt changes in logits.
2. We greatly appreciate your suggestion to include an ablation study on "how dynamic soft modulation prevents abrupt changes in logits.". In Appendix D, we have included the ablation study for soft modulation and provided cases analysis, as shown in Table 8 and Figure 13. The ablation results based LLaVA-1.5 and Qwen-VL are as below:

Method | CHAIRs | CHAIRi |
-- | -- | -- |
LLaVA-1.5 (DeCo) | 37.8 | 11.1
LLaVA-1.5 (DeCo wo/ max_probs) | 41.2 | 11.6
Qwen-VL (DeCo) | 42.2 | 10.7
Qwen-VL (DeCo wo/ max_probs) | 45.8 | 12.3

Overall, soft modulation preserves the original generation style to some extent, thereby preventing incoherent semantics and hallucinations caused by abrupt changes in logits.
3. The sequence is generated using the POPE method, where the first token is either "yes" or "no," typically serving as the model’s answer. Since the generation method aligns with image description, we can influence the model's output through constrained decoding.

If you have any further questions on OpenReview, we would be happy to provide detailed responses.

**Comment by sunnychenxiwang** [comment]
hi, do you have any further questions?

**Comment by WangYipu2002** [comment]
Thank you for your reply, I have no further questions

**Comment by WangYipu2002** [comment]
Hi，If I want to evaluate some newer models, such as Qwen2.5-VL, llava-onevision. But the transformers version in your codebase is relatively old and incompatible with some new models, is there a better solution to this?

**Comment by sunnychenxiwang** [comment]
Unfortunately, at the moment, there is no other solution besides modifying the file transformers/generation/utils.py in the latest transformers versions adapted for these MLLMs.

**Comment by WangYipu2002** [comment]
thank you. Can you tell me which version of transformers you are using? I'm guessing 4.29?

**Comment by sunnychenxiwang** [comment]
Yes, we refer to the version of the transformer(4.29.2) used in OPERA.
